topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,718 @@
+> {: .solution }
+{: .question}
+
+> ### {% icon hands_on %} (Optional) Hands-on: IP strength estimation (other samples)
+>
+> Run the same analysis on the other ChIP-seq data along with their corresponding input and compare the output
+{: .hands_on}
+
+# Step 4: Normalization
+
+One of the goals in ChIP-seq data analysis is finding regions on the genome which are enriched for the ChIP data of interest (regions with significantly higher read coverage for the ChIP data comparing to its corresponding input). In the following exercise we would like to know where the H3K4me3 sites are. For this we need to extract which parts of the genome have been enriched (i.e. more reads mapped) within the samples that underwent immunoprecipitation. However, to reach a reliable comparison the data needs to be normalized to remove any technical bias. For the normalization we have two steps:
+
+1. Normalization by sequencing depth
+2. Normalization by input file
+
+To learn how to do the normalization, we will take the `wt_H3K4me3_rep1` sample as ChIP data and `wt_input_rep1` as input.
+
+## Generation of coverage files normalized by sequencing depth
+
+> ### {% icon hands_on %} Hands-on: Estimation of the sequencing depth
+>
+> 1. **IdxStats** {% icon tool %} with the following parameters
+>    - {% icon param-files %} *"BAM file"*: `wt_H3K4me3_rep1.bam` and `wt_input_rep1.bam`
+>
+> > ### {% icon question %} Questions
+> >
+> > 1. What is the output of **IdxStats** {% icon tool %}?
+> > 2. How many reads has been mapped on chrX for the input and for the ChIP-seq samples?
+> > 3. Why are the number of reads different? And what could be the impact of this difference?
+> >
+> > > ### {% icon solution %} Solution
+> > > 1. This tool generates a table with 4 columns:  reference sequence identifier, reference sequence length, number of mapped reads and number of placed but unmapped reads. Here it estimates how many reads mapped to which chromosome. Furthermore, it tells the chromosome lengths and naming convention (with or without 'chr' in the beginning)
+> > 2. 1,204,821 for ChIP-seq samples and 1,893,595 for the input
+> > 3. The number of reads can be different because of  different sequencing depth. It can bias the interpretation of the number of reads mapped to a specific genome region and the identification of the H3K4me3 sites. Specially here, as the number of reads for the input is higher than the ChIP data less regions could be identified having a significantly higher read coverage for the ChIP data comparing to the corresponding input.
+> > {: .solution }
+> {: .question}
+{: .hands_on}
+
+The different samples have usually a different sequencing depth, i.e. a different number of reads. These differences can bias the interpretation of the number of reads mapped to a specific genome region. We first need to make the samples comparable by normalizing the coverage by the sequencing depth.
```

Comment:
The answer to part 3. and this paragraph sound like a repetition and are imho to equal.